src/style.css

```diff
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.searchBar {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: flex;
+  width: 100%;
+  margin: 20px 0;
+  height: 70px;
+  padding: 25px 0 25px 62px;
+}
+
+.searchBar--small {
+  height: 42px;
+  padding: 10px 0 11px 33px;
+}
+
+.searchBar::placeholder {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.searchBar--small::placeholder {
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.searchBar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchBar:focus {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 800;
+  outline: none;
+  color: #3d4e61;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.search {
+  position: relative;
+}
+
+.img {
+  position: absolute;
+  top: 25px;
+  left: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.img--small {
+  position: absolute;
```

Comment:
```suggestion
```

Reply:
after I have fixed all, the tests are failed 
![image](https://user-images.githubusercontent.com/52597158/205371342-53aee0cf-1606-45ff-939a-2d92c4610a71.png)